server/testserver.go

```diff
@@ -83,6 +85,11 @@ func (ts *TestServer) Start() error {
 	if ts.Addr == "" {
 		ts.Addr = "127.0.0.1:0"
 	}
+	if ts.CertDir == "" {
+		// Always load server certs. Not sure if this path is a good assumption though,
+		// something relative might be best.
+		ts.CertDir = path.Join(os.ExpandEnv("$GOPATH"), "src/github.com/cockroachdb/cockroach/resource/test_certs")
```

Comment:
Changed to Getenv.
I was looking into using the embedded data, and will probably do this with the further certs change.